cookbooks/bcpc/templates/default/carbon.conf.erb

```diff
@@ -92,6 +109,32 @@ USE_FLOW_CONTROL = True
 # By default, carbon-cache will log every whisper update. This can be excessive and
 # degrade performance if logging on the same volume as the whisper data is stored.
 LOG_UPDATES = False
+LOG_CACHE_HITS = False
```

Comment:
That would mean that config.py will be examined for defaults, and keep in mind it is always subject to change at the whim of other developers and maintainers.  As is I am not a big fan of implicit declarations.